py/private/py_binary.bzl

```diff
@@ -205,7 +214,7 @@ py_binary = rule(
 py_test = rule(
     doc = "Run a Python program under Bazel. Most users should use the [py_test macro](#py_test) instead of loading this directly.",
     implementation = py_base.implementation,
-    attrs = py_base.attrs,
+    attrs = py_base.attrs | py_base.test_attrs,
```

Comment:
huh, I didn't know about this merge operator on structs, neat

Reply:
oh actually the spec says 

> - x | y means either x or y.

Should this be merging them instead?

Reply:
This is for merging dictionaries, the right takes precedence but the result is otherwise the union